src/api/general.ts

```diff
@@ -132,6 +132,29 @@ export class General {
     return view<T>({ aptosConfig: this.config, ...args });
   }
 
+  /**
+   * Queries for a Move view function with JSON, this provides compatability with the old `aptos` package
+   * @param args.payload Payload for the view function
+   * @param args.options.ledgerVersion The ledger version to query, if not provided it will get the latest version
+   *
+   * @example
+   * const data = await aptos.view({
+   *  payload: {
+   *   function: "0x1::coin::balance",
+   *   typeArguments: ["0x1::aptos_coin::AptosCoin"],
+   *   functionArguments: [accountAddress],
```

Comment:
`accountAddress` is of type `AccountAddress` - you would need to do `accountAddress.toString()`